tests/phpunit/class-wc-stripe-connect-test.php

```
@@ -206,6 +211,76 @@ public function test_save_stripe_keys_decommissions_previous_webhook_before_savi
+		}
+
+		// App OAuth live connection so refresh_connection() proceeds past its guard.
+		WC_Stripe_Helper::update_main_stripe_settings(
+			[
+				'testmode'        => 'no',
+				'publishable_key' => 'pk_live_123',
+				'secret_key'      => 'sk_live_123',
+				'connection_type' => 'app',
+				'refresh_token'   => 'rt_live_123',
+			]
+		);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'test_refresh_connection_stops_retrying_on_terminal_error'. (undefined)

(StaticAccess)